src/main/resources/interface-specification.yml

```diff
@@ -7446,34 +7446,37 @@ paths:
               schema:
```

Comment:
@nttdata-rtorsoli could you please re-arrange endpoints to avoid the noise?

Reply:
I am not be able to restore the previous list of endpoints

Reply:
perfect 👌